src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/SubjectAccessRequestServiceTest.kt

```diff
@@ -0,0 +1,212 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.integration
+
+import org.junit.jupiter.api.Test
+import org.skyscreamer.jsonassert.JSONAssert
+import org.skyscreamer.jsonassert.JSONCompareMode
+import org.springframework.beans.factory.annotation.Autowired
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1ApplicationTimelinessCategory
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.factory.PersonRisksFactory
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.ApprovedPremisesApplicationStatus
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.ApprovedPremisesType
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.RiskTier
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.RiskWithStatus
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.subjectaccessrequest.SubjectAccessRequestService
+import java.time.LocalDate
+import java.time.OffsetDateTime
+
+class SubjectAccessRequestServiceTest : IntegrationTestBase() {
```

Comment:
I wonder if we'll want to break this into CAS1/CAS2 and CAS3 tests to keep it manageable? I guess we can see how it looks as we add more tests

Reply:
yeah - i'm thinking of refactoring the tests a bit as we go, particularly around the stub data.  agree this would be a good idea to break into separate sections.